### ISSUE TITLE
split update-claim into precise actions

### ISSUE THREAD

**Issue by maxlath** [inventaire/inventaire#98]
I'm considering splitting `update-claim` into `add-claim`, `update-claim`, and `remove-claim`, as the [current implementation deduces the action from the existing parameters](https://github.com/inventaire/inventaire/blob/403889ef3085eabbd271de867b9fe35a1e7cac30/server/models/entity.coffee#L94-L109), which can thus have undesired consequences if a parameter name happen to be misspelled.

**Comment by thewholelifetolearn** [comment]
I'm not sure that separating the actions is a good idea. That would mean that you should know before if you are adding or updating.  
After all, al actions are a kind of update. You update from *null* to `something`, from `something` to `something else` and from `something else` to *null*.  
Just submitting one value is fine. Don't need the old one.

**Comment by maxlath** [comment]
I think having all in one endpoint is (slightly) more convenient but more error prone, especially for edition from the command line where it's easy to make a typo and thus trigger the wrong action, possibly even without noticing it: I would prefer having a more explicit syntax which would reject anything that doesn't look as expected.

You do need the old one for both *from `something` to `something else`* and *from `something else` to `null`.*

**Comment by jum-s** [comment]
it's true that `update-claim` is a too-many-instructions function that you have to go all the way through if you ask why [this](https://github.com/inventaire/inventaire/blob/403889ef3085eabbd271de867b9fe35a1e7cac30/server/models/entity.coffee#L81)
No matter what, you create an entity & destroy it, I know beforehand those two actions to me
So, yeah max kill some logic here (only if necessary for future potential actions) it might ends with more readable code, to me

**Comment by maxlath** [comment]
@jum-s true, even if I consider that the API clearness and ease of use have priority over internal code readabillity. Fortunately, in this case, I think both hint for the same direction.
